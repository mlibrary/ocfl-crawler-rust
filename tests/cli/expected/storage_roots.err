tests/cli/1.2 is not a storage root
