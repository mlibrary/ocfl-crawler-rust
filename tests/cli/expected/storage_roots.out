storage root: tests/cli/1.0
object root: tests/cli/1.0/minimal_one_version_one_file--tests/cli/1.0/minimal_no_content
storage root: tests/cli/1.1
object root: tests/cli/1.1/minimal_one_version_one_file--tests/cli/1.1/minimal_no_content
